### PR TITLE
0.2.0

### PR DESCRIPTION
**Pull request by rawgroundbeef**
- Ditched eslint in favor of standard.
- Removed full lodash dependency in favor of individually packaged functions.
- Moved Joi from deps to peerDeps
- Proposed moving from JoifulForm, JoifulInput to simply Form, Input
- Removed unneeded re-renders due to unneeded setStates.
- Only return values that are in the schema when sending formValues through onSubmit.

### CODE REVIEW

**Comment by rawgroundbeef** [comment]
@kjanoudi please review.

File: src/Input.js [hunk 1]
```diff
@@ -90,19 +90,11 @@ export default class Input extends Component {
...
-      if (!get(this, ['context', 'form', 'elemTypes', is])) {
-        return `
-          [JoifulReactForms Error] The requested input type of
-          ${is} does not have a defined element type
-        `
-      }
-
-      if (is === 'select' && get(schema, '_valids._set', []).length === 0) {
-        return `
-          Warning! ${name} is a select element but no valid params were provided.
```
Comment (kjanoudi) [inline src/Input.js]: was this error no longer needed?

**Comment by kjanoudi** [comment]
hard to tell exactly what changed with the diffs including the standard stuff

**Comment by rawgroundbeef** [comment]
Shoot. You're right. Sorry about that.
- Form's onChange and onSubmit logic improved.
  - onChange had 3 setStates, sometimes firing more than once per change, causing re-renders.
  - onSubmit passes formValues filtered based on the schema. So if somehow your Form's value property had some extra keys, the form only return's what it knows is in your schema.. this way you don't have to check again onSubmit.
- Changing to standard lint setting, I think makes things easier to understand. Though, it ruins the diff, hopefully you agree.
- Proposing an api change from `{ JoifulForm, Joifulnput }` to simply `{ Form, Input }`. It's less typing and how often are you in a situation where you have more than one `<Form>` or `<Input>`?
- Lightened dependencies by using separate lodash packages vs the whole lodash library and moving joi to peerDependencies.

**Pull request merged**